app/routes/errors.py

```diff
@@ -27,7 +30,7 @@
 errors_blueprint = Blueprint("errors", __name__)
 
 
-def log_error(error, status_code):
+def log_exception(exception, status_code):
```

Comment:
Not a blocking comment, but this shouldn't be specific to the routes module, it's more of a utility method

Reply:
And, given that we use structlog, tx_id should be bound to the root logger for the duration of the request context... thereby rendering this superflous given we have the `logger.exception` method